src/extension.ts

```diff
@@ -201,6 +202,7 @@ async function _activateExtension(
 
   context.subscriptions.push(
     uriHandler,
+    WebsocketManager.getInstance(),
```

Comment:
websocket manager has a dispose() method which will clean up and close the websocket.